internal/expr/parser.go

```diff
@@ -99,16 +107,23 @@ func (p *Parser) add(part queryPart) {
 			&bypassPart{p.input[p.prevPart:p.partStart]})
```

Comment:
Can we change this to use the newly created function `p.addRemainder()`?

Reply:
I've changed it. Might as well reuse the code. I've left the commend though to make it clear what is happening.